cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -121,8 +122,9 @@
               "  1. User and System Datasets (upgrades the coprocessor jars)\n" +
               "  2. Upgrade Schedule Triggers\n" +
               "  3. Stream State Store\n" +
-              "  4. Updates system metadata for all existing entities\n" +
-              "  5. Removes any metadata that may have left behind for deleted datasets.\n" +
+              "  4. Workflow run records in Application Metadata Store\n" +
```

Comment:
I see, that makes sense.  Points 2, 5, and 6 have a verb as the first word so it is very inconsistent.  Perhaps we should remove 'Upgrade' and 'Updates' from those lines?